client/src/screens/Patient/Form/details.js

```diff
@@ -63,7 +63,7 @@ const FormsDetails = (props) => {
           {!!data
             && data.length
             ? data.map((row) => (
-              <StyledTableRow key={row.created}>
+              <StyledTableRow key={`${row.created}_${row.title}`}>
```

Comment:
Don't we have `id` ? If this records in coming from database then let me know I will change it so that you can get `id` as well.

Reply:
At first I look for Id, if Id is not present then I will have to do this way. Please add  Id in the forms records.